src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -320,6 +395,9 @@ def get_vector_count_query(self):
             'timeout': '5s'
         }
 
+    def _is_large_int(value, low_threshold=2 ** 31, high_threshold=2 ** 64):
+        return isinstance(value, int) and low_threshold <= abs(value) <= high_threshold
```

Comment:
This threshold is different from the thresholds in `_verify_numerical_field_value`. Dose that mean the `_verify_numerical_field_value` is not useful anymore?

Reply:
Thanks for the clarification!